app/templates/index.hbs

```diff
@@ -57,3 +60,51 @@
     </div>
   </div>
 </main>
+
+<AuModal
+  @title="Geef een naam en beschrijving"
+  @modalOpen={{this.showModal}}
+  @closeModal={{this.closeModal}} as |Modal| >
+  <Modal.Body>
+    <div class="au-o-grid au-o-grid--small">
+      <div class="au-o-grid__item au-1-1">
```

Comment:
Nitpick, but it seems the grid is only used to provide some spacing between the input and the textarea. I think it would be better to either use the [`AuFormRow` setup](https://appuniversum.github.io/ember-appuniversum/?path=/story/patterns-forms--default-form) or the [flow utils](https://appuniversum.github.io/ember-appuniversum/?path=/story/objects-flow--page) instead.